web_favicon/views/templates.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<openerp>
+    <data>
+        <template id="layout" inherit_id="web.layout">
+            <xpath expr="//link[@rel='shortcut icon']" position="replace">
+                <link rel="shortcut icon" href="/web_favicon/favicon" />
```

Comment:
Prefer to use `rel="icon"`([source](https://mathiasbynens.be/notes/rel-shortcut-icon))